src/main/java/com/diluv/schoomp/Webhook.java

```diff
@@ -95,6 +120,11 @@ public Response sendMessage (Message message) throws IOException {
         
         // Encodes the message object as JSON.
         final String encoded = GSON.toJson(message);
+    
+        if (debugMode) {
+            System.out.println("Encoded message:");
+            System.out.println(encoded);
```

Comment:
Is Log4J or SLF4J available in this project? I don't like using system.out in production.